internal/pkg/scorecard/scorecard_output.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"encoding/json"
+	"fmt"
+	schelpers "github.com/operator-framework/operator-sdk/internal/pkg/scorecard/helpers"
+	scapi "github.com/operator-framework/operator-sdk/pkg/apis/scorecard"
+	scapiv1alpha1 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha1"
+	"io/ioutil"
+)
+
+func printPluginOutputs(version string, pluginOutputs []scapiv1alpha1.ScorecardOutput) error {
+
+	var list scapi.ScorecardFormatter
+	var err error
+	list, err = combinePluginOutput(pluginOutputs)
+	if err != nil {
+		return err
+	}
+
+	if schelpers.IsV1alpha2(version) {
+		list = scapi.ConvertScorecardOutputV1ToV2(list.(scapiv1alpha1.ScorecardOutput))
+	}
+
+	// produce text output
+	if scViper.GetString(OutputFormatOpt) == TextOutputFormat {
```

Comment:
Nit: This is out of scope for this PR, but I'm not a huge fan of the global variables used throughout scorecard (e.g. `scViper`) because they cause non-obvious side effects if you're looking at the call site of the functions that use the global variables.

In a follow-on, I think it would be worth it to refactor things so that viper is used only in the `cmd/operator-sdk/scorecard` package to parse the config from flags, env, config files, etc. and then populate a scorecard struct with the resulting config, so that we don't have to constantly interrogate a global viper instance.

Reply:
agreed, I could write up a new Jira story for refactoring that.

Reply:
https://jira.coreos.com/browse/OSDK-586

Reply:
https://jira.coreos.com/browse/OSDK-586

another one that falls into the refactoring camp that would be an improvement.